kennethwu42/PolyEats
# Image upload

**Description**
This pull request, fulfills Cypress testing and has a happy path that starts from login/signup and goes through most if not all buttons within the app.

**Testing/Changes** 
Testing was done manually first, then making code on a cypress testing file that recreates what a human user would do.

**Testing Steps**
Be sure to have "npm install cypress --save-dev" instaleld in the root directory. Then start both front and backend via respective npm run commands. Then in the root directory run npx cypress open. The provided files should already be set up so you can test and watch an automated user goes through the site.

**Additional Notes:**
Let me know if there is anything else that needs to be added to the test cases.
